mcan/src/bus.rs

```diff
@@ -388,6 +388,14 @@ impl<Id: mcan_core::CanId, D: mcan_core::Dependencies<Id>, C: Capacities> Can<'_
         while self.internals.can.cccr.read().init().bit() != init {}
     }
 
+    /// Re-enters "Normal Operation" if in "Software Initialization" mode.
+    /// In Software Initialization, messages are not received or transmitted.
+    /// Configuration cannot be changed. In Normal Operation, messages can
+    /// be transmitted and received.
+    pub fn enter_operational_mode(&mut self) {
```

Comment:
There's no need to "enter operational mode" from Can that is already in operational mode. If the purpose of this method is to "reset" Can if it ended up in the error passive mode, then I think it should be renamed AND available on CanInternals or sth. Having it on the `Can` limits how and where it can be used.

Reply:
> There's no need to "enter operational mode" from Can that is already in operational mode.

The way I see it, anything with INIT=1 is non-operational. Then it is not known statically whether the bus is operational or not. Do you draw the line in a different place (maybe CCE=1) when you say it is already operational?

> If the purpose of this method is to "reset" Can if it ended up in the error passive mode,

Yes, the use case is to restore a peripheral that has disabled itself to working order, but error passive is not the same as INIT=1 and is not cleared by setting INIT=0. Only things like bus Bus_Off or uncorrected bit errors in message RAM automatically set INIT=1. 

> it should be renamed

If "operational" is ambiguous, do you have any suggestions for what it should be named? Would it help if the name was more aligned with the user's manual, e.g. `enter_normal_operation`, or are you looking for something more along the lines of `reset` or `restore`?

> available on CanInternals

Makes sense. I'll change it.

Reply:
Right, I forgot about the property of the MCAN that it also physically reenters INIT state upon bus-off condition. Then naming of the method is probably fine, as long as it is available on the `CanInternals` struct.

Considering that the MCAN state is not statically known, maybe it'd make sense to expose some API that'd allow a user to inspect it? Not sure how exactly we can check the state of the transceiver but that'd probably neat thing to have

Reply:
The events that cause INIT=1 can be observed through interrupts (`BusOff` and `UncorrectedBitError`), but I agree that it would be good to be able to inspect directly.

I think we should leave transceiver management out of scope for mcan, at least for now.

Reply:
We agreed to move this change to #11, resolving.